cloud/user-management.md

```diff
@@ -0,0 +1,53 @@
+# User management
+You can manage users for your Timescale Cloud account by navigating to the
+`Members` section in your Cloud console. From here, you can see your FIXME
+
+## About Timescale Cloud user management
+Timescale Cloud allows you to collaborate with other users on your projects.
+When you have created your project, you can add other users so that they can see
+your project in their Timescale Cloud console. You can add, manage, and delete
+users from the `Members` section of the Cloud console.
+
+If you are the creator of the Project, you are assigned an Administrator role,
+and have the ability to add, manage, and delete other users. If you have been
+added to a project, you are assigned a Member role.
+
+## Add or remove members
+You can add members to your project group from the `Members` section in your
+Cloud console.  The new member does not need  to have a Timescale Cloud account
+before you add them. If they do not yet have an account, they are prompted to
+create one.
+
+<procedure>
+
+### Adding members to a project group
+1.  [Log in to your Timescale Cloud account][cloud-login] and navigate to
+    the `Members` section.
+1.  Click `Add New Member`.
+1.  Type the email address of the person that you want to add. A confirmation
+    email is sent to the email address you entered.
+1.  Follow the instructions in the confirmation email to confirm the email
+    address. Alternatively, the new member can create or sign in to a Timescale Cloud account with the email address, and accept the invitation by navigating to the `Invitations` section, and clicking `Accept`.
+1.  The new member can switch to the shared project by selecting it from the
+    `Project` drop down menu.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/FIXME.png" alt="Adding a new member to a project in Timescale Cloud"/>
+
+</procedure>
+
+<procedure>
+
+### Removing members from a project group
```

Comment:
A section called `Leaving a project` can also be made. 

For members that are not the owner/admin of a project, they have the ability to remove themselves from it via the "Leave project" button inside of the Members page. This button will only appear for non-admin members.

Reply:
I've written the procedure, but it doesn't look like it's active yet.

Reply:
This feature should be active on dev now. It's located bottom right in the member's page.

Reply:
Yep, I have it active and could add and remove members, but couldn't see how to leave a project. I don't seem to get the `Leave Project` button.

Reply:
So you can't leave your own project but you can leave projects other people invite you to. That may be the issue. I just invited you to a project. Try and leave that.

Reply:
Yeah, that's what I was trying to do. I don't see the button.

Reply:
@Loquacity Is the “Leave project” button still not appearing for you?

Reply:
I'll check again today.

Reply:
So I can't click the Accept button at all today, I can only decline an invitation 😢

Reply:
Ah ha! I have it now! 🎉